addon/webextension/selector/ui.js

```diff
@@ -582,7 +588,17 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       this.bgRight.style.width = "100%";
 
       if (this.downloadNotice) {
-        this.downloadNotice.style.bottom = (docHeight - pageYOffset - winBottom + 10) + "px";
+        this.downloadNotice.style.top = (pageYOffset + winBottom - 60) + "px";
+
+        const repositionDownloadNotice = () => {
```

Comment:
The download notice should be tied to the browser window, but b/c it is injected into an iframe that's the size of the document we have to do some jujitsu to make it stick